examples/src/test/scala/com/gu/tableversions/examples/DatePartitionedTableLoaderSpec.scala

```diff
@@ -42,6 +44,8 @@ class DatePartitionedTableLoaderSpec extends FlatSpec with Matchers with SparkHi
     implicit val tableVersions: TableVersions[IO] = InMemoryTableVersions[IO].unsafeRunSync()
     implicit val metastore: Metastore[IO] = new SparkHiveMetastore[IO]()
 
+    spark.sparkContext.hadoopConfiguration.set("fs.versioned.baseFS", "file")
+
```

Comment:
I think we should encapsulate this in a function in `VersionedFilesystem`, e.g. `setUnderlyingScheme` or something. I.e. I don't think the client code should have to fiddle with hadoop config directly.